spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/controller/SkipperController.java

```diff
@@ -47,7 +47,7 @@
  * @author Ilayaperumal Gopinathan
  */
 @RestController
-@RequestMapping("/")
+@RequestMapping("/api")
```

Comment:
Ideally, we can use `@BasePathAwareController` but there seems to an issue with mapping the `basePath` into underlying method request mappings.
Also, using `@RequestMapping("${spring.data.rest.base-path:/api}")` has issues outside the servlet context.